baseclasses/utils/tecplotIO.py

```diff
@@ -1154,8 +1154,8 @@ def _readZoneHeader(self, lines: List[str], iCurrent: int) -> Tuple[Dict[str, An
         headerString = ", ".join(header)
 
         # Use regex to parse the header information
-        zoneNameMatch = re.search(r'zone t\s*=\s*"([^"]*)"', headerString, re.IGNORECASE)
-        zoneName = zoneNameMatch.group(1) if zoneNameMatch else None
+        zoneNameMatch = re.search(r'(zone t)\s*=\s*[\'""]?([^\'""\n,]+)[\'""]?(?=[,\n]|$)', headerString, re.IGNORECASE)
```

Comment:
For future people's sake, can you add a comment here explaining what this regex matches?

Reply:
Yeah I need to review and add more tests for the regex patterns. To my knowledge, there's no established rules published by Tecplot that describe the formatting of the ASCII header data. I'm considering simpler approach that removes the regex and just assumes all data follows an `=` sign terminates with either a `comma` or `newline` unless it's surrounded by single or double quotes.